### DISCUSSION TITLE
How to run map locally

### DISCUSSION THREAD

**Discussion by dancesWithCycles** [piratenpanda/veggiekarte#128] [General]
Hi folks,
Thank you so much for providing and maintaining this map. Appreciated!

I am active in developing and maintaining the ```Der Gruene Kompass``` map.

https://greenwire.greenpeace.de/greenpeace-braunschweig/inhalt/der-gruene-kompass

http://umap.openstreetmap.fr/de/map/der-grune-kompass-braunschweig_93968#12/52.2641/10.5211

I am searching for a new back and front end for this map. This way I stumbled over veggiekarte.de.

I am studying veggiekarte now to learn how it is working and to decide, if it might be a candidate for a new front end of ```Der Gruene Kompass```.

I did not find any information yet on how to run this map on my local development host. ```npm run start``` definitely does not suit this purpose. ;-)

I am also interested how to you update this map to the latest changes made in OSM.

I am appreciating any hints that brings light into darkness. ;-)

Cheers!

**Comment by piratenpanda** [comment]
Hi,
just clone the repo and open index.html in a browser. New data is acquired by running the refresh.py script. On the server this is done via cron. That's it basically.

**Reply by piratenpanda** [reply]
It's really tailored for this specific purpose so I don't know how feasible it is to change it to something completely different though
